bin/grouper-api

```diff
@@ -1,31 +1,30 @@
 #!/usr/bin/env python
 
 import argparse
-from contextlib import closing
 import logging
 import os
 import sys
+from contextlib import closing
```

Comment:
nit: we started sorting alphabetically in more recent diffs. so 

```
from contextlib import closing
import logging
import os		  import os
import sys		  import sys
```

Reply:
ah, fair enough. I'll reorder accordingly.